### PR TITLE
Nadia branch

### PR DESCRIPTION
**Pull request by npittigl**
Review js to merge to development branch

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        // target HTML to append checkboxes
+        const cuisineChoices = document.querySelector('.cuisineChoices');
+
+        // create div element to populate with label and input
+        const optionButton = document.createElement('div');
+        optionButton.classList.add('optionButton');
+
+        // add input/label HTML to div
+        optionButton.innerHTML =
+            `<input type="checkbox" id="cuisineOption" name="cuisineOption" value="${cuisine}">
```
Comment (GHJBaek) [inline script.js]: would need to specify which label links to which input

Comment (npittigl) [reply script.js]: added index as parameter to .forEach and included it in the innerHTML to distinguish input/labels more clearly (ex. id="cuisineOptions${index}").

**Review by GHJBaek — APPROVED** [review]
LGTM

**Review by PamLu3 — APPROVED** [review]
good to go

**Pull request merged**